examples/expression-bodied-functions/expression_bodied_functions.description

```diff
@@ -0,0 +1,2 @@
+// A function body can be a single expression. Such a function is called an expression-bodied function and the
```

Comment:
We haven't used the term `expression-bodied function` in the spec. Shall we avoid this term?

Reply:
Picked up the term from https://github.com/ballerina-platform/ballerina-spec/issues/395. If not, we don't have a simple enough term to refer to it right?